src/scenes/PlantsDashboardRouter/components/PlantDashboardMap.tsx

```diff
@@ -618,6 +622,35 @@ const PlantDashboardMap = ({
     strings,
   ]);
 
+  const nameTags = useMemo((): MapNameTag[] | undefined => {
+    return plantingSites
+      .map((site): MapNameTag | undefined => {
+        if (site.boundary) {
+          const points = site.boundary.coordinates
```

Comment:
If this site to nameTags gets used anymore, it might be worth moving the code to a reusable helper function where you just pass in the list of planting sites